src/java/org/apache/cassandra/io/sstable/format/big/BigTableScanner.java

```diff
@@ -117,7 +117,7 @@ private BigTableScanner(SSTableReader sstable,
         this.listener = listener;
     }
 
-    private static List<AbstractBounds<PartitionPosition>> makeBounds(SSTableReader sstable, Collection<Range<Token>> tokenRanges)
+    public static List<AbstractBounds<PartitionPosition>> makeBounds(SSTableReader sstable, Collection<Range<Token>> tokenRanges)
```

Comment:
This is a bit of a hack, and there is probably a better way to do this, just added this to have a mechanism to infer unrepaired bytes in sstables.